servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/WorkItem.java

```diff
@@ -3,66 +3,56 @@
 
 package com.microsoft.azure.servicebus.primitives;
 
-import java.time.*;
-import java.util.concurrent.*;
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ScheduledFuture;
 
-class WorkItem<T>
-{
+class WorkItem<T> {
     private final TimeoutTracker tracker;
     private final CompletableFuture<T> work;
     private ScheduledFuture<?> timeoutTask;
     private Exception lastKnownException;
 
-    public WorkItem(final CompletableFuture<T> completableFuture, final Duration timeout)
-    {
+    WorkItem(final CompletableFuture<T> completableFuture, final Duration timeout) {
         this(completableFuture, TimeoutTracker.create(timeout));
     }
 
-    public WorkItem(final CompletableFuture<T> completableFuture, final TimeoutTracker tracker)
-    {
+    WorkItem(final CompletableFuture<T> completableFuture, final TimeoutTracker tracker) {
         this.work = completableFuture;
         this.tracker = tracker;
     }
 
-    public TimeoutTracker getTimeoutTracker()
-    {
+    public TimeoutTracker getTimeoutTracker() {
```

Comment:
Will you have to remove this `public` modifier too like you had to for the ctors? Or is this in a future PR?